sklearn/utils/_show_versions.py

```diff
@@ -70,49 +72,22 @@ def get_version(module):
     return deps_info
 
 
-def _get_blas_info():
-    """Information on system BLAS
-
-    Uses the `scikit-learn` builtin method
-    :func:`sklearn._build_utils.get_blas_info` which may fail from time to time
-
-    Returns
-    -------
-    blas_info: dict
-        system BLAS information
-
-    """
-    from .._build_utils import get_blas_info
-
-    cblas_libs, blas_dict = get_blas_info()
-
-    macros = ['{key}={val}'.format(key=a, val=b)
-              for (a, b) in blas_dict.get('define_macros', [])]
-
-    blas_blob = [
-        ('macros', ', '.join(macros)),
-        ('lib_dirs', ':'.join(blas_dict.get('library_dirs', ''))),
-        ('cblas_libs', ', '.join(cblas_libs)),
-    ]
-
-    return dict(blas_blob)
-
-
 def show_versions():
     "Print useful debugging information"
 
     sys_info = _get_sys_info()
     deps_info = _get_deps_info()
-    blas_info = _get_blas_info()
 
     print('\nSystem:')
     for k, stat in sys_info.items():
         print("{k:>10}: {stat}".format(k=k, stat=stat))
 
-    print('\nBLAS:')
-    for k, stat in blas_info.items():
-        print("{k:>10}: {stat}".format(k=k, stat=stat))
-
     print('\nPython deps:')
     for k, stat in deps_info.items():
         print("{k:>10}: {stat}".format(k=k, stat=stat))
+
+    print('\nNumpy BLAS/LAPACK:')
+    numpy.show_config()
+
+    print('\nScipy BLAS/LAPACK:')
+    scipy.show_config()
```

Comment:
This prints half a page of text, we don't really want it in issue reports I think? Also, wouldn't it suffer from the same limitations as  the previous `_get_blas_info` ?

Reply:
I think `show_config` is more accurate in general. It's based on infos gathered when building numpy, whereas the previous `_get_blas_info` takes its infos from `numpy.distutils.system_info.get_info` which gather infos at run time.

But I'm really not sure how both work so I might be completely wrong...